lib/engine/game/base.rb

```diff
@@ -384,6 +384,13 @@ def init_optional_rules(optional_rules)
         optional_rules
       end
 
+      def check_optional_rules!
+        min_players = @players.size
+        max_players = @players.size
+        error_string = meta.check_options(@optional_rules, min_players, max_players)&.[](:error)
```

Comment:
@roseundy I added this to the `setup` so that for 1822PNW I didn't need to duplicate the optional rule compatibility logic in the meta and the game class. Is this useful for 1825? Or should I move this out of the base game and only into `G1822PNW`?

Reply:
I'd leave it in the base, the way you have it. I do dislike that 1825 is checking options in multiple places.